Library/Homebrew/cask/blacklist.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Cask
+  module Blacklist
+    WHITELISTED_CASKS = %w[
+      adobe-acrobat-pro
+      adobe-acrobat-reader
+      adobe-air-sdk
+      adobe-air
+      adobe-connect
+      adobe-creative-cloud-cleaner-tool
+      adobe-creative-cloud
+      adobe-digital-editions
+      adobe-dng-converter
+      adobe-lens-profile-creator
+    ].freeze
+
+    def self.blacklisted_reason(name)
+      return if WHITELISTED_CASKS.include?(name)
+
+      case name
+      when /^adobe\-.+/i
+        "Adobe casks were removed because they are too difficult to maintain."
+      when /^audacity$/i
```

Comment:
The `/i` is redundant. Cask names are always lowercase, and that might even already be checked by `audit`.